woocommerce-services.php

```diff
@@ -769,10 +771,116 @@ public function show_tos_notice() {
 			<?php
 		}
 
+		public function reinstall_notice() {
+			if ( get_option( 'wc_connect_dismiss_deprecation', false ) ) {
+				return;
```

Comment:
I don't think we want to make this notice dismissable. I would just keep nagging the merchant, there's absolutely no reason to not upgrade.